compiler/cpp/src/audit/t_audit.cpp

```diff
@@ -41,12 +41,12 @@ void thrift_audit_warning(int level, const char* fmt, ...) {
 
 void thrift_audit_failure(const char* fmt, ...) {
   va_list args;
-  fprintf(stderr, "[Thrift Audit Failure:%s] ", g_curpath.c_str());
+  //fprintf(stderr, "[Thrift Audit Failure:%s] ", g_curpath.c_str());
   va_start(args, fmt);
   vfprintf(stderr, fmt, args);
   va_end(args);
   fprintf(stderr, "\n");
-  g_return_failure = true;
+  //g_return_failure = true;
 }
 
```

Comment:
We cannot just undo others' warnings and exit value manipulation...
I imagine there are some difficulties to keep these ?